overlord/ifacestate/ifacemgr.go

```diff
@@ -137,6 +138,20 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 	return m, nil
 }
 
+// AppArmorPromptingRunning returns true if prompting is running.
+func (m *InterfaceManager) AppArmorPromptingRunning() bool {
+	return m.useAppArmorPrompting
+}
+
+// Allow m.UseAppArmorPrompting to be mocked in tests
+var assessAppArmorPrompting = func(m *InterfaceManager) bool {
+	return m.assesAppArmorPrompting()
+}
+
+func (m *InterfaceManager) InterfacesRequestsManager() *apparmorprompting.InterfacesRequestsManager {
```

Comment:
this needs a doc comment and should also mention that it can be used only after StartUp